client/server/pages/test/index.js

```diff
@@ -165,7 +165,7 @@ const buildApp = ( environment ) => {
 			serverRender,
 		} = require( 'calypso/server/render' );
 		mocks = { ...mocks, attachBuildTimestamp, attachI18n, attachHead, renderJsx, serverRender };
-		mocks.sanitize = require( 'sanitize' );
+		mocks.sanitize = require( 'calypso/server/sanitize' );
```

Comment:
Same import as the code being tested (https://github.com/Automattic/wp-calypso/blob/trunk/client/server/pages/index.js#L21)